File: Python/Mhs.py
# delkarasi class
# class mahasiswa berisi data mahasiswa berupa nama, nim, prodi, dan fakultas
class Mhs:
    # private
    __nama = ""
    __nim = ""
    __prodi = ""
    __fakultas = ""

    # public
    # constructor
    def __init__(self, nama="", nim="", prodi="", fakultas=""):
        self.__nama = nama
        self.__nim = nim
        self.__prodi = prodi
        self.__fakultas = fakultas

    # setter dan getter
    def get_nama(self):
        return self.__nama

    def get_nim(self):
        return self.__nim

    def get_prodi(self):
        return self.__prodi

    def get_fakultas(self):
        return self.__fakultas

    def set_nama(self, nama):
        self.__nama = nama

    def set_prodi(self, prodi):
        self.__prodi = prodi

    def set_nim(self, nim):
        self.__nim = nim

    def set_fakultas(self, fakultas):
        self.__fakultas = fakultas

File: C++/Mhs.cpp
// import library
#include <iostream>
#include <string>
#include <list>

// standard namespace
using namespace std;

// class mahasiswa berisi data mahasiswa berupa nama, nim, prodi, dan fakultas
class Mhs
{
    // atribut private
private:
    string nama;
    string nim;
    string prodi;
    string fakultas;
    // atribut public
public:
    // Constructor untuk mengubah value menjadi default
    Mhs()
    {
        this->nama = "";
        this->nim = "";
        this->prodi = "";
        this->fakultas = "";
    }
    // Public method
    // setter dan getter
    string get_nama()
    {
        return this->nama;
    }
    string get_nim()
    {
        return this->nim;
    }
    string get_prodi()
    {
        return this->prodi;
    }
    string get_fakultas()
    {
        return this->fakultas;
    }

    void set_nama(string nama)
    {
        this->nama = nama;
    }
    void set_nim(string nim)
    {
        this->nim = nim;
    }
    void set_prodi(string prodi)
    {
        this->prodi = prodi;
    }
    void set_fakultas(string fakultas)
    {
        this->fakultas = fakultas;
    }
    // destructor
    ~Mhs()
    {
    }
};